src/domain/registration/form/RegistrationForm.tsx

```diff
@@ -192,13 +194,18 @@ const RegistrationForm: FunctionComponent = () => {
                     <Icon src={happyAdultIcon} className={styles.childImage} />
                     <h2>{t('registration.form.guardian.info.heading')}</h2>
                   </div>
-                  <div className={styles.guardianEmail}>
-                    <label>
-                      {t('registration.form.guardian.email.input.label')}
-                    </label>
-                    <p>{values.guardian.email}</p>
-                  </div>
-
+                  <EnhancedInputField
+                    id="guardian.email"
+                    name="guardian.email"
+                    required={true}
+                    validate={validateEmail}
+                    label={t('registration.form.guardian.email.input.label')}
+                    component={InputField}
+                    placeholder={t(
+                      'registration.form.guardian.email.input.placeholder'
+                    )}
+                    value={values.guardian.email}
```

Comment:
Are you sure that `formik` doesn't handle the value field for you automatically? By default it at least attempts to apply handlers and value based on the name prop.